lib/icingadb/icingadb.hpp

```diff
@@ -57,6 +57,13 @@ class IcingaDB : public ObjectImpl<IcingaDB>
 		std::mutex m_Mutex;
 	};
 
+	enum StateUpdate
+	{
+		Volatile    = 1ull << 0,
+		RuntimeOnly = 1ull << 1,
```

Comment:
Btw. any special reason for **ull**?

Reply:
It's not necessary in this case. I just got used to always writing it this way as it's safe up to `1ull << 63`. This is especially useful if you shift by a variable because then the compiler usually won't warn you about `1<<n` as it would with `1<<42`.

Reply:
I'd make the class enum inheriting uint_fast8_t.

Reply:
> I'd make the class enum inheriting uint_fast8_t.

Is this a request for change or just an "I'd have it done differently"? So can I merge it as is?

Reply:
Requests for changes are red, not green :)